requirements/frozen_docs.py2.txt

```diff
@@ -91,7 +90,6 @@ oauthlib==2.1.0
 os-service-types==1.3.0
 oslo.i18n==3.22.1
 oslo.utils==3.37.1
-ovirt-engine-sdk-python==4.3.0
```

Comment:
I'm wondering if it really not necessary

Reply:
it is still there - https://github.com/ManageIQ/wrapanapi/blob/master/requirements.txt#L15